### PR TITLE
Increase the test resource-group-controller memory request to 200Mi

### PR DESCRIPTION
**Pull request by nan-yu**
The resource-group-controller's default memory request is increased from 50Mi to 200Mi to address the crashlooping issue on the autopilot cluster.

### CODE REVIEW

**Comment by nan-yu** [comment]
This PR includes changes from https://github.com/GoogleContainerTools/kpt-config-sync/pull/63. This PR needs rebase after https://github.com/GoogleContainerTools/kpt-config-sync/pull/63 is merged.

/hold

File: manifests/test-resources/resourcegroup-manifest.yaml [hunk 1]
```diff
@@ -503,7 +503,7 @@ spec:
         resources:
           requests:
             cpu: 100m
-            memory: 50Mi
+            memory: 200Mi
```
Comment (sdowell) [inline manifests/test-resources/resourcegroup-manifest.yaml:506]: /lgtm
/hold

**Review by sdowell — APPROVED** [review]

**Review by janetkuo — APPROVED** [review]
/lgtm

**Pull request merged**